vendor/assets/stylesheets/rolodex/settings/utilities/lib/_clearfix.sass

```diff
@@ -0,0 +1,29 @@
+// Clearfix
+// ========================================
+
+.clearfix:before,
+.clearfix:after,
+.group:before,
+.group:after
+  content: " "
```

Comment:
group is a standard rolodex set. however, search the bellycard repos and you'll find tons of instances of `.clearfix`. supporting both.